corehq/ex-submodules/casexml/apps/case/views.py

```diff
@@ -55,6 +55,8 @@ def dynamic_properties(self):
         if self.case.location_id:
             dynamic_data['location_id'] = self.case.location_id
 
+        dynamic_data['case_name'] = self.case.name
```

Comment:
Does name always have a value? Would it make sense to make this conditional?
```suggestion
        if self.case.name:
            dynamic_data['case_name'] = self.case.name
```

Reply:
Even if it doesn't have a value we still want to add it otherwise you wouldn't be able to set the name of a case without one.